server.js

```diff
@@ -33,6 +33,7 @@ const app = express();
 app.use(cors()); // enables the server to talk to local things
 const GEOCODE_API_KEY = process.env.GEOCODE_API_KEY;
 const WEATHER_API_KEY = process.env.WEATHER_API_KEY;
+const TRAIL_API_KEY = process.env.TRAIL_API_KEY;
```

Comment:
Putting your API keys to the top is smart, I put mine in each individual function.